managed/src/main/java/com/yugabyte/yw/controllers/CloudProviderController.java

```diff
@@ -159,6 +163,11 @@ public Result create(UUID customerUUID) throws IOException {
         String hostedZoneId = provider.getHostedZoneId();
         switch (provider.code) {
           case "aws":
+            CloudAPI cloudAPI = cloudAPIFactory.get(provider.code);
+            if (cloudAPI != null && !cloudAPI.isValidCreds(config, formData.get().region)) {
```

Comment:
Move the region to the formData.

Reply:
It's already taken care of. Updated the summary as well. Thanks!